backoffice/templates/backoffice/user.html

```diff
@@ -162,6 +155,16 @@ <h3>{% trans "Enrollments" %}</h3>
 
         <div class="col-md-12">
 
+            {% if userform.instance.is_active and not userform.instance.is_superuser and not userform.instance.is_staff %}
+                <form action="{% url 'masquerade-mask' %}" method="POST" target="_blank" style="display: inline">
+                    {% csrf_token %}
+                    <input type="hidden" name="mask_user" value="{{ userform.instance.username }}">
+                    <button class="btn btn-info" type="submit">{% trans "Impersonate" %}</button>
+                </form>
+
+                <a class="btn btn-warning" onclick="$.get({% url 'masquerade-unmask' %});">{% trans "Stop impersonation" %}</a>
+            {% endif %}
+
             <button class="btn btn-warning" data-toggle="modal" data-target="#change-password">
                 {% trans "Change password" context "change-user-password" %}
             </button>
```

Comment:
Edge case: On line 172, we may want to add a verification, because a superuser could ban himself viewing its own user page